safe-global/safe-react
# feat: notification centre popper

## What it solves
Adds a popper under the notificatin bell.

## How this PR fixes it
A popper containing a list of all cached notifications is displayed `onClick` of the notification bell in the header.

Note: this does not include the required actions section yet as that will be added during the notification clarification https://github.com/safe-global/safe-react/issues/3951.

## How to test it
Click on the bell in the header and observe the new popper. Attempt to sign a transaction and see the dispatched notification now accessible via the popper

## Screenshots
![notification centre](https://user-images.githubusercontent.com/20442784/173594521-42be28a6-2f5f-46e0-8943-adb8f780d17d.gif)
